torch/_inductor/quantized_lowerings.py

```diff
@@ -144,11 +157,24 @@ def int4pack_mm_cpu(
                 (mat1, mat2, group_size, qScaleAndZeros), aten_layout
             ).output_node()
 
+        # define functions to generate example inputs for weight and group size
+        # otherwise, autotuner generates example inputs of all zeros for them
+        def get_example_weight(x: torch._inductor.ir.IRNode) -> torch.Tensor:
+            shape = x.get_size()
+            device = x.get_device()
+            return torch.randint(0, 255, shape, dtype=torch.uint8, device=device)
```

Comment:
don't we need to account for strides here?  This assumes contiguous.

Can we share some code with the other example input generation?

Reply:
Thanks for your comments.
- Now I have added a check: if weight is not contiguous, we won't take GEMM template as a choice for max-autotuning. And I have also added an assertion inside `get_example_weight`.
- As for the second question, I have searched in the PyTorch source code and didn't find any code that can be shared. This case is special because weight's dtype is uint8.

Thanks.